qualang_tools/optimal_weights/ge_discriminator_train_n_benchmark.py

```diff
@@ -0,0 +1,157 @@
+# $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$ #
```

Comment:
need to remove this file

Reply:
The example files needs to be in qua_libs.
The py_qua_tools is luke numpy, it doesn't have "example" files. People will not be able to see them.